settings/movement.py

```diff
@@ -0,0 +1,169 @@
+from data.movement import MovementActions
+from memory.space import Allocate, Bank, Reserve, Read
+import instruction.asm as asm
+
+WALK_SPEED = 2
+SPRINT_SPEED = 3
+DASH_SPEED = 4
+
+class Movement:
+    def __init__(self):
+        import args
+        self.movement = args.movement or MovementActions.AUTO_SPRINT
+
+        if self.movement != MovementActions.ORIGINAL:
+            self.mod()
+
+    def mod(self):
+        length = 0
+        src = []
+
+        (length, src) = self.get_auto_sprint_src()
+
+        space = Allocate(Bank.F0, length, "Sprint subroutine")
+        space.write(src)
+
+        src = [
+            asm.JSL(space.start_address_snes),
+        ]
+        space = Reserve(0x04e21, 0x04e37, "auto sprint", asm.NOP())
+        space.write(src)
+
+        self.sliding_dash_fix()
+
+
+    def get_auto_sprint_src(self):
+        import args
+        CURRENT_MAP_BYTE = 0x82 # 2 bytes
+        OWZERS_MANSION_ID = 0x0D1 # the door room can create visual artifacts on the map while dashing
```

Comment:
I think this should be the Owzer's Basement map which, if I'm looking at https://everything8215.github.io/ff6tools/ff6tools.html correctly, should be 207 (0x0cf)

Reply:
I enabled this and confirmed it should be 0xCF.

![image](https://user-images.githubusercontent.com/96998881/166614229-630dd250-850e-47bf-affc-2ccc8d538c73.png)